templatestore/views.py

```diff
@@ -424,13 +417,77 @@ def get_template_versions_view(request, name):
             status=404,
         )
 
+@csrf_exempt
+def get_tiny_url_from_db(request, name, version):
+    if request.method != "GET":
+        return HttpResponseBadRequest("invalid request method: " + request.method)
+    try:
+        templateTable=Template.objects.get(name=name)
+        try:
+            versionTable= TemplateVersion.objects.get(template_id_id=templateTable.id, version=version)
+            data = json.dumps(versionTable.tiny_url)
+            return HttpResponse(data)
+        except Exception:
+            raise (
+                Exception(
+                    "Validation: Template Version with version " + version + " does not exist"
+                )
+            )
+    except Exception:
+        raise (
+            Exception(
+                "Validation: Template with name " + name + " does not exist"
+            )
+        )
+    
+@csrf_exempt
+def save_tiny_url(request):
+    if request.method != "PUT":
```

Comment:
is save_tiny_url an update operation?
if not, this should not be PUT

Reply:
Yes, it is an update operation.